src/game/overlay.c

```diff
@@ -243,17 +244,19 @@ void Overlay_DrawHealthBar()
     int32_t show = g_HealthBarTimer > 0 || hit_points <= 0
         || g_Lara.gun_status == LGS_READY;
     switch (g_Config.healthbar_showing_mode) {
+    case T1M_BSM_FLASHING_OR_DEFAULT:
+        show |= hit_points <= (LARA_HITPOINTS * 20) / 100;
+        break;
+    case T1M_BSM_FLASHING_ONLY:
+        show = hit_points <= (LARA_HITPOINTS * 20) / 100;
+        break;
     case T1M_BSM_ALWAYS:
         show = 1;
         break;
     case T1M_BSM_NEVER:
         show = 0;
         break;
-    case T1M_BSM_FLASHING_OR_DEFAULT:
-        show |= hit_points <= (LARA_HITPOINTS * 20) / 100;
-        break;
-    case T1M_BSM_FLASHING_ONLY:
-        show = hit_points <= (LARA_HITPOINTS * 20) / 100;
+    default:
```

Comment:
This `default: break` idiom is to avoid the compiler complaining about the switch missing certain cases right?

Reply:
I don't think there's a compiler error for no default, but IMO it makes it clear that the default case is just `show` and we didn't forget about it.